cloudsync/tasks.py

```diff
@@ -191,7 +192,7 @@ def schedule_retranscodes(self):
             ])
     except:  # pylint: disable=bare-except
         error = "schedule_retranscodes threw an error"
-        log.exception(error)
+        log.error(error)
```

Comment:
I had to change `exception` to `error` in commit 6ccbfb6fbd5ea2a7810d5e0101f5667a5c7a8903 because `structlog-sentry` raises an `AttributeError` when it tries to use `getattr()` to get `logging.Logger`'s log method. (See commit comment.)

Reply:
I think adding `wrapper_class=structlog.stdlib.BoundLogger,` fixes that problem?

Reply:
I'll change it and see. Does `exception()` do anything that `error()` with a stacktrace doesn't do? I had checked the `logging` documentation and it seemed to say it was just `error()` with a stacktrace.

Reply:
log.exception should include the stacktrace automatically, so you don't have to include `exc_info=True`)

Reply:
Changed at https://github.com/mitodl/odl-video-service/pull/865/commits/dca212eeda1dd02a88bff98cba93f18493a1f16b#diff-0da2c698f19bf424e9fdcd1e851eda69R195

Reply:
Right, the fact that `error()` doesn't include the stacktrace by default is why I had `exc_info=True` in there. I had switched to it because I was getting an AttributeError with `exception()`. But I think that besides adding a stacktrace `exception()` isn't any different than `error()`. If the currently-running CI builds succeed then we're fine. If we keep getting AttributeError I'll need to change back to `error()`. Fortunately, the exception will always be passed to Sentry with BoundLogger, but BoundLogger only affects Sentry's behavior. It won't make sure that the exception gets into the log message that gets shipped to ELK.